build.sbt

```diff
@@ -23,9 +23,9 @@ publishArtifact in Test := false
 pomIncludeRepository := { x => false }
 
 libraryDependencies ++= Seq(
-  "io.swagger" % "swagger-core" % "1.5.8",
-  "org.scalatest" %% "scalatest" % "2.2.5" % "test",
-  "com.fasterxml.jackson.module" %% "jackson-module-scala" % "2.4.2",
```

Comment:
Swagger-core has a dependency on the older jackson-databind and there are compatibility issues with the newer Jackson versions. I opened a pull request on Swagger-core and saw some issues with upgrading Jackson.